http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/HttpRoutes.scala

```diff
@@ -62,7 +62,7 @@ class HttpRoutes(
         complete(StatusCodes.InternalServerError -> report)
       case e: Throwable =>
         //NOTE: this needs SprayJsonSupport._, ErrorReportJsonSupport._, and errorReportSource all imported to work
-        complete(StatusCodes.InternalServerError -> ErrorReport(e))
+        complete(StatusCodes.InternalServerError -> ErrorReport(e.getMessage, Some(StatusCodes.InternalServerError), ErrorReport.causes(e), Seq(), Some(e.getClass)))
```

Comment:
this ErrorReport required some digging since its overloaded a bunch, but this should correspond to the signature
` def apply(message: String, statusCode: Option[StatusCode], causes: Seq[ErrorReport], stackTrace: Seq[StackTraceElement], exceptionClass: Option[Class[_]])(implicit source: ErrorReportSource): ErrorReport =
    ErrorReport(source.source, message, statusCode, causes, stackTrace, exceptionClass)`